test/java/seedu/addressbook/util/TypicalPersons.java

```diff
@@ -0,0 +1,82 @@
+package seedu.addressbook.util;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.Address;
+import seedu.addressbook.data.person.Email;
+import seedu.addressbook.data.person.Name;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.Phone;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.person.UniquePersonList;
+import seedu.addressbook.data.tag.Tag;
+import seedu.addressbook.data.tag.UniqueTagList;
+
+/**
+ *
+ */
+public class TypicalPersons {
+
+    public static Person johnDoe, selbyWhite, betsyCrowe, steveSam;
+    
+    private static void initialisePersons() {
+        try {
+            johnDoe = new Person(new Name("John Doe"),
+                                 new Phone("98765432", false),
+                                 new Email("johnd@gmail.com", false),
+                                 new Address("John street, block 123, #01-01", false),
+                                 new UniqueTagList(Collections.emptySet()));
+            selbyWhite = new Person(new Name("Selby White"),
+                                    new Phone("111344", false),
+                                    new Email("me@haoo.com", true),
+                                    new Address("UK", false),
+                                    new UniqueTagList(Collections.emptySet()));
+            betsyCrowe = new Person(new Name("Betsy Crowe"),
+                                    new Phone("1234567", true),
+                                    new Email("betsycrowe@gmail.com", false),
+                                    new Address("Newgate Prison", true),
+                                    new UniqueTagList(new Tag("friend"), new Tag("criminal")));
+            steveSam = new Person(new Name("Steve Sam"),
+                                  new Phone("1231", true),
+                                  new Email("ab@apple.com", true),
+                                  new Address("Silicon Vally", true),
+                                  new UniqueTagList(new Tag("boss")));
+        } catch (IllegalValueException e) {
+            e.printStackTrace();
+            assert false : "not possible";
+        }
+    }
```

Comment:
That class is mostly stable now, so you can copy-paste-modify that code into your and continue with your PR. If that PR is merged first, you'll have to update your branch and de-conflict your class a bit at that point.